concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,6 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Josserand-1983-188	Josserand, Judy Katherine	1983		118	basic, areal	Spanish, English	Mixtec languages		Josserand1983	Josserand1983	This concept list consists of 188 cognate sets [Josserand (1983)](:bib:Josserand1983) used for subgrouping. The cognate sets cover 120 varieties (coverage differs across cognate sets) and a proto-Mixtec reconstruction for each concept. Tones are not marked.	489-678	
+Duerr-1987-110	Dürr, Michael	1987		110	basic, areal	English	Mixtec languages		Durr1987	Duerr1987	This concept list consists of 110 cognate sets across 17 varieties of Mixtec including a proto-Mixtec reconstruction for each. It was used by [Dürr (1987)](:bib:Durr1987) for tone reconstruction. It overlaps to some degree with Josserand 1983.	38-55	
```

Comment:
Great, additional "" are fixed! We usually add new concept lists at the end of the file.

Note that the number of items differs in Josserand (188 in the title of the list and 118 in the column `ITEMS`.

Reply:
ugh! oops! the right number is 188. fixed this and moved the two to the end